apps/hash-api/src/graph/knowledge/system-types/linear-integration-entity.ts

```diff
@@ -349,24 +352,79 @@ export const linkIntegrationToWorkspace: ImpureGraphFunction<
       ],
     });
   } else {
-    await createLinkEntity<SyncLinearDataWith>(context, authentication, {
-      webId: extractWebIdFromEntityId(linearIntegrationEntityId),
-      properties,
-      linkData: {
-        leftEntityId: linearIntegrationEntityId,
-        rightEntityId: workspaceEntityId,
+    /**
+     * Allow the user creating the link, the Linear bot and the web admins to administer the link (e.g. to delete it),
+     * and allow other web members to view the link.
+     */
+    const linearIntegrationRelationships: EntityRelationAndSubjectBranded[] = [
+      {
+        relation: "administrator",
+        subject: {
+          kind: "account",
+          subjectId: authentication.actorId,
+        },
       },
-      entityTypeIds: [
-        systemLinkEntityTypes.syncLinearDataWith.linkEntityTypeId,
-      ],
-      relationships: [
-        ...createDefaultAuthorizationRelationships(authentication),
-        {
-          // Allow the system account ID to view the link
-          relation: "viewer",
-          subject: { kind: "account", subjectId: systemAccountId },
+      {
+        relation: "administrator",
+        subject: {
+          kind: "account",
+          subjectId: linearBotMachineId,
         },
-      ],
+      },
+      {
+        relation: "setting",
+        subject: {
+          kind: "setting",
+          subjectId: "administratorFromWeb",
+        },
+      },
+      {
+        relation: "setting",
+        subject: {
+          kind: "setting",
+          subjectId: "viewFromWeb",
+        },
+      },
+    ];
+
+    const linearIntegrationWebId = extractWebIdFromEntityId(
+      linearIntegrationEntityId,
+    );
+
+    const linkEntity = await createLinkEntity<SyncLinearDataWith>(
+      context,
+      authentication,
+      {
+        webId: linearIntegrationWebId,
+        properties,
+        linkData: {
+          leftEntityId: linearIntegrationEntityId,
+          rightEntityId: webEntityId,
+        },
+        entityTypeIds: [
+          systemLinkEntityTypes.syncLinearDataWith.linkEntityTypeId,
+        ],
+        relationships: linearIntegrationRelationships,
+      },
+    );
+
+    const linkEntityUuid = extractEntityUuidFromEntityId(
+      linkEntity.metadata.recordId.entityId,
+    );
+
+    await createPolicy(context.graphApi, authentication, {
+      name: `linear-integration-bot-view-link-entity-${linkEntityUuid}`,
+      principal: {
+        type: "actor",
+        actorType: "machine",
+        id: linearBotMachineId,
+      },
+      effect: "permit",
+      actions: ["viewEntity"],
+      resource: {
+        type: "entity",
+        id: linkEntityUuid,
+      },
     });
```

Comment:
I guess this could be a general policy in https://github.com/hashintel/hash/blob/a94b7dad2f2844965674c9b845bb1eb4522bfbdb/libs/%40local/graph/postgres-store/src/store/postgres/seed_policies.rs#L135-L139
Above, there is a similar example for the google bot, which has access to `@google`'s `account` type: https://github.com/hashintel/hash/blob/a94b7dad2f2844965674c9b845bb1eb4522bfbdb/libs/%40local/graph/postgres-store/src/store/postgres/seed_policies.rs#L107-L133

But we can also easily do that in a follow-up.

Reply:
Yes, let's talk about how best to deal with these bots in follow-up